starter-code/src/main.js

```diff
@@ -42,9 +42,14 @@ $(document).ready(function(){
   // Add all the div's to the HTML
   document.getElementById('memory_board').innerHTML = html;
   // Bind the click event of each element to a function
-$('.back').on('click', function () {
-   
+$('.back').on('click', function () { 
+
+$(this).toggleClass('back');
+$(this).next().addClass('back');
+
+if (this.checkIfPair){
```

Comment:
CAUTION!!! This 'this' is not your memoryGame, then you can't not use its methods. This 'this' is the jQuery 'this', pointing to the div you clicked. It is important to understand this.

Furthermore, checkIfPair is a method, you have to use parentheses to invoke it. Again, here won't work.